src/main/java/com/savvato/tribeapp/controllers/PermissionsAPIController.java

```diff
@@ -60,7 +78,7 @@ public ResponseEntity<Boolean> addPermissions(@RequestBody @Valid PermissionsReq
             return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(rtn);
         }
     }
-    @DeleteMapping
+    @RequestMapping(value = { "/api/permissions" }, method= RequestMethod.DELETE)
```

Comment:
The endpoint mapping before is the correct way of doing it (DeleteMapping). Just FYI, we made a system wide change to that way of doing things a couple weeks ago.